### ISSUE TITLE
Bug in regex "velociraptor/artifacts/definitions/Windows/System/HostsFile.yaml"

### ISSUE THREAD

**Issue by aspir** [Velocidex/velociraptor#914]
instead of:
```
"^\\s+(?P<Resolution>[^\\s]+)\\s+" +
                "(?P<Hostname>[^\\s]+)\\s*" +
                "(?P<Comment>#*.*)$"
```
which assume there must be whitespace key in the beginning of each line. I believe it suppose to be like this:
```
"^\\s*(?P<Resolution>[^\\s]+)\\s+" +
                "(?P<Hostname>[^\\s]+)\\s*" +
                "#*\\s*(?P<Comment>.*)$"
```
Which supports whitespace(not must) in the beginning and not showing '#' char in the command result field

**Comment by scudette** [comment]
This artifact could probably use a test too. According to https://en.wikipedia.org/wiki/Hosts_(file)#File_content each line may contain more than one name as well, so this does not work either. An example file:

```
   # Comment 
# Comment 2
127.0.0.1  localhost loopback  # More comments
::1        localhost
```

**Comment by mgreen27** [comment]
Just reviewing this now. I think we can fix the first condition easily but will need to think how to do the duplicate host names.

**Comment by scudette** [comment]
I think this is fixed in 0.5.7? @mgreen27 ?

**Comment by mgreen27** [comment]
Yes it is but I cant close this one.
